pages/index.html

```diff
@@ -152,26 +152,24 @@ <h3 class="usa-font-lead">A few samples of the web design standards at work:</h3
     </div>
   </div>
   <div class="usa-grid">
-    <div class="usa-width-five-sixths">
-      <h3 class="usa-example-heading usa-link-external">
-        <a class="usa-example-link" href="https://vote.usa.gov/">
-          vote.usa.gov
-        </a>
-      </h3>
-      <img class="usa-img-example" src="{{ site.baseurl }}/assets-styleguide/img/home/example_voteUSAgov.png" alt="vote.usa.gov homepage">
-      <h3 class="usa-example-heading usa-link-external">
-        <a class="usa-example-link" href="{{ site.baseurl }}/assets-styleguide/img/home/example_VAappealsmock.png">
-        Mockup of VA Appeals Modernization Screen
-        </a>
-      </h3>
-      <img class="usa-img-example" src="{{ site.baseurl }}/assets-styleguide/img/home/example_VAappealsmock.png" alt="VA Appeals Modernization Screen">
-      <h3 class="usa-example-heading usa-link-external">
-        <a class="usa-example-link" href="https://playbook.cio.gov">
-          U.S. Digital Service Playbook
-        </a>
-      </h3>
-      <img class="usa-img-example" src="{{ site.baseurl }}/assets-styleguide/img/home/example_playbook.png" alt="U.S. Digital Service Playbook homepage">
-    </div>
+    <h3 class="usa-example-heading usa-link-external">
+      <a class="usa-example-link" href="https://vote.usa.gov/">
+        Vote.usa.gov
```

Comment:
@angel I think this should be lowercase. See @awfrancisco's edits to the home page.